client/src/assets/icons/green-pass.tsx

```diff
@@ -1,20 +1,24 @@
 import React from 'react';
 import { useTranslation } from 'react-i18next';
 
-function GreenPass(
-  props: JSX.IntrinsicAttributes & React.SVGProps<SVGSVGElement>
-): JSX.Element {
+interface GreenPassProps
+  extends JSX.IntrinsicAttributes,
+    React.SVGProps<SVGSVGElement> {
+  hushScreenReaderText?: boolean;
+}
+function GreenPass(props: GreenPassProps): JSX.Element {
   const { t } = useTranslation();
-
+  const { hushScreenReaderText = false, ...rest } = props;
   return (
     <>
       <svg
-        aria-label={t('icons.passed')}
+        {...(hushScreenReaderText && { 'aria-hidden': true })}
+        {...(!hushScreenReaderText && { 'aria-label': t('icons.passed') })}
```

Comment:
Question: Should this be on the parent interactive element (`Link`/`a` tag)?

In which case, this should always be `aria-hidden=true`

Reply:
In the beta rwd, this icon is used in the toggle buttons for the courses. But we don't want the icon to have the default `aria-label` text in this case because the completion status of the course is being added after the course name. So when we add the GreenPass icon with the `hushScreenReaderText` flag then not only do we not add an `aria-label` but we just hide it completely. Otherwise, for all of the other course listings, the GreenPass icon will work exactly the same as it always.